### PR TITLE
fix ML

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

File: shed_tool_conf.xml [hunk 1]
```diff
@@ -16647,8 +16647,6 @@
             <id>toolshed.g2.bx.psu.edu/repos/devteam/bowtie2/bowtie2/2.3.2.2</id>
             <version>2.3.2.2</version>
         </tool>
-    </section>
-    <section id="mapping" name="Mapping" version="">
```
Comment (wm75) [inline shed_tool_conf.xml:16651]: Still hoping to get this scripted :) but yes, shrinking the file is good.

**Review by wm75 — APPROVED** [review]
LGTM

**Comment by bgruening** [comment]
We just need to check the tool.yaml are correct.

**Comment by wm75** [comment]
yeah, on it. ecology.yaml is what puts the ML stuff into statistics :(

**Pull request merged**